apps/src/lib/ui/SchoolInfoInterstitial.jsx

```diff
@@ -196,7 +197,54 @@ export default class SchoolInfoInterstitial extends React.Component {
     };
   }
 
+  isBlank(field) {
+    // return true when field is blank
+    return !!(!field || field.trim() === '');
+  }
+
+  validateSubmission = () => {
+    const {country, schoolType, schoolName, ncesSchoolId} = this.state;
+    let errors = {};
+    let isValid = true;
+
+    if (this.isBlank(country)) {
+      errors.country = true;
+      isValid = false;
+    } else {
+      if (this.isBlank(schoolType)) {
+        errors.schoolType = true;
+        isValid = false;
+      } else {
+        if (country === 'United States') {
+          const ncesSchoolType = ['public', 'private', 'charter'];
```

Comment:
Can you add a comment here about what's special about "ncesSchoolType"? If I were just coming into this file without working on a bigger project around school info, I might not know what this means.